pkg/lru/cache.go

```diff
@@ -0,0 +1,24 @@
+package lru
```

Comment:
nit: I think the standard in our repo is to have `cache.go` for interface and `cache_impl.go` for implementation